ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -941,42 +941,6 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: rosa-aws-sts-shared-vpc
-- as: aws-rosa-sts-hcp-full-f360
```

Comment:
Any reason to remove the 4.16 jobs?

Reply:
Added 4.16 jobs again

Reply:
Shall we update the title to reflect the real change ?

Reply:
Sure, done